evalml/automl/automl_search.py

```diff
@@ -143,6 +145,7 @@ def search(
             in time series problems, values should be passed in for the time_index, gap, forecast_horizon, and max_delay variables.
         n_splits (int): Number of splits to use with the default data splitter.
         verbose (boolean): Whether or not to display semi-real-time updates to stdout while search is running. Defaults to False.
+        timing (boolean): Whether or not to display pipeline search times to stdout. Defaults to False.
```

Comment:
Nitpicky clarification: logging info is not guaranteed to display that information in stdout, that will only happen if the logging level is set low enough to expose it. It'd be more accurate to say:
```
Whether or not to write pipeline search times to the logger. Defaults to False.
```

By default, if `timing` is set to True, the user would still not see the timings being logged in stdout since the default logging behavior is at the `warning` level (will not show info/debug level logs). To see this, they would either need to configure the logger themselves, or set `verbose=True`. Alternatively, they can choose to dump the log into a file or otherwise redirect that info, in which case it would be logged somewhere but not appearing in stdout.

Sorry for the info dump, I'm just intimately familiar with our logging behavior 😅

Reply:
It's always good for me to learn more so "info dump(s)" are always great.